perses/tests/test_topology_proposal.py

```diff
@@ -336,6 +337,71 @@ def test_mutate_from_alanine():
         else:
             _ = generate_dipeptide_top_pos_sys(ala.topology, amino, ala.system, ala.positions, system_generator, conduct_geometry_prop=False)
 
+def test_protein_atom_maps():
+    
+    # Get alanine dipeptide in vacuum test system
+    ala, system_generator = generate_atp()
+    
+    
+    # Define function for checking that the atom map is correct
+    def check_atom_map(topology_proposal, reference_map):
```

Comment:
can this be a utility outside the `test` fn?

Reply:
It will never be used outside of this function, so there's no need to move it outside `test_protein_atom_maps`